src/platform/AT32/target/AT32F435G/target.h

```diff
@@ -22,7 +22,7 @@
 #pragma once
 
 #ifndef TARGET_BOARD_IDENTIFIER
-#define TARGET_BOARD_IDENTIFIER "A43G"
+#define TARGET_BOARD_IDENTIFIER "A435"
```

Comment:
Actually there is no reason to use only 4 :) I had a look - you could make it `A435G` and `A435M`